Sources/Shared/Models/Album.swift

```diff
@@ -86,8 +87,9 @@ import Foundation
     @objc public var videoThumbnails: NSArray?
     @objc public var user: VIMUser?
     @objc public var theme: String?
-    @objc dynamic private var metadata: [String: Any]?
-    @objc dynamic private var connections: [String: Any]?
+    @objc private var metadata: [String: Any]?
```

Comment:
is there a reason why we had these set as `dynamic` before?

Reply:
Not a good reason, no. I had a discussion with Alex about this a few months ago and it was likely a mistake. I couldn't think of a good reason why only these two properties would need to be dynamically dispatched when everything else isn't, yet all of them are accessible via Objective-C. 

A lot of our models are marked with `dynamic` which feels like maybe a mistake? Or do we really want to make sure we're using dynamic instead of static dispatch?

Reply:
yeah I can't really think of a reason this would be necessary.